modules/oadp-release-notes-1-1-2.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-2_{context}"]
+= OADP 1.1.2 release notes
+
+The OADP 1.1.2 release notes include product recommendations, a list of fixed bugs and descriptions of known issues.
+
+== Product recommendations
+
+To prepare for the upgrade from VolSync 0.5.1 to 0.6.0, in the `openshift-adp` namespace you must add this annotation.
+
+----
+$ oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'
+----
+
+== Fixed bugs
+
+The following bugs have been fixed in this release:
+
+* link:https://issues.redhat.com/browse/OADP-1150[OADP-1150]
+* link:https://issues.redhat.com/browse/OADP-290[OADP-290]
+* link:https://issues.redhat.com/browse/OADP-1056[OADP-1056]
+
+== Known issues
+
+This release has the following known issues:
+
+* OADP currently does not support backup and restore of AWS EFS volumes using restic in Velero (link:https://issues.redhat.com/browse/OADP-778[*OADP-778*]).
+
+* CSI backups might fail due to a Ceph limitation of `VolumeSnapshotContent` snapshots per PVC.
++
+You can create many snapshots of the same persistent volume claim (PVC) but cannot schedule periodic creation of snapshots:
++
+** For CephFS, you can create up to 100 snapshots per PVC.
+** For RADOS Block Device (RBD), you can create up to 512 snapshots for each PVC. (link:https://issues.redhat.com/browse/OADP-804[*OADP-804*]) and (link:https://issues.redhat.com/browse/OADP-975[*OADP-975*])
```

Comment:
Do these linked issues apply to only this bullet?

Reply:
Corrected